griptape/labs/tools/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .zoom.tool import Zoom
 from .openai_dalle.tool import OpenAiDallE
 from .sql_redshift.tool import SqlRedshiftClient
+from .proxycurl.tool import ProxycurlClient
```

Comment:
Let's call this package `proxycurl_client`, so it's consistent with other package names.